### ISSUE TITLE
Missing binary for Ubuntu 22.04

### ISSUE THREAD

**Issue by elissonmichael** [zakird/wkhtmltopdf_binary_gem#142]
```
Invalid platform, must be running on Ubuntu 16.04/18.04/20.04 CentOS 6/7/8, Debian 9/10, archlinux amd64, or intel-based Cocoa macOS (missing binary
```
I would like to comment that the [Ubuntu 22.04 download](https://wkhtmltopdf.org/downloads.html) is already available:

**Comment by davidBentoPereira** [comment]
I had to used this gem instead : https://github.com/rposborne/wkhtmltopdf-heroku to make it work with Ubuntu 22.04.
But the problem :
- My CI Server is on Ubuntu 20.04
- My local environement is on MacOS arm64
- My Production Server is on Ubuntu 22.04

So now I have to use one gem for my local and CI environement, and another for my production environment. That's a nice mess 😅

**Comment by leesmith** [comment]
@davidBentoPereira Sounds like a proper mess! How'd you configure it to handle all those envs?

**Comment by davidBentoPereira** [comment]
@leesmith I put one gem  in the production group of the Gemfile and the other in the other groups 😅
I don't have a better solution for now. Until wkhtmltopdf can support Ubuntu 22.4 on his own 😙

**Comment by javiervg** [comment]
> @leesmith I put one gem in the production group of the Gemfile and the other in the other groups sweat_smile I don't have a better solution for now. Until wkhtmltopdf can support Ubuntu 22.4 on his own kissing_smiling_eyes

Hello, FYI, it is already supported, https://github.com/wkhtmltopdf/wkhtmltopdf/issues/5189

We got it to work with wicked_pdf gem, download the binary directly (sudo apt-get -y install wkhtmltopdf) and updating the wicked_pdf.rb initializer as recommended in the documentation of wicked_pdf

WickedPdf.config = {
  exe_path: '/usr/bin/wkhtmltopdf',
  enable_local_file_access: true
}

**Comment by roostinghawk** [comment]
Waiting for this PR's approve
https://github.com/zakird/wkhtmltopdf_binary_gem/pull/141

**Comment by franzliedke** [comment]
This can be closed, now that #150 has been merged and released.

**Comment by unixmonkey** [comment]
Thanks for pointing this out. Closing. Please reopen if you have trouble with this specific version.
